HacktoberfestProject.Web/Models/UserinfoViewModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections.Generic;
+

```

Comment:
These seem like a duplication of the DTO models. For such a small project I don't think we need both DTOs and ViewModels.

Reply:
Currently, they are. Most important reason to do so is to not disclose TableEntities to the frontend because they have fields that are not interesting. Or you maybe even do not want to disclose (like ETag). But I'm fine with using the entities if that's preferred.